README.md

```diff
@@ -1,145 +1,62 @@
-    <li><a href="#contact">Contact</a></li>
-    <li><a href="#acknowledgments">Acknowledgments</a></li>
-  </ol>
-</details>
-
-<!-- ABOUT THE PROJECT -->
-
-## About The Project
-
-Description of project
-
-<p align="right">(<a href="#top">back to top</a>)</p>
-
-### Built With
-
--   [React](https://reactjs.org/)
--   [Bootstrap](https://getbootstrap.com)
--   [NodeJS](https://nodejs.org/en)
-
-<p align="right">(<a href="#top">back to top</a>)</p>
-
-### Prerequisites
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md)
 
-### Installation
-
-1. Contact christopher.hancock@seattlecolleges.edu for project secret keys
-2. Create and navigate into a directory
-    ```sh
-     mkdir club_website
-     cd club_website
-    ```
-3. Clone the repo
-    ```sh
-     git clone https://github.com/NSC-Computer-Science-Club/club_website.git
-    ```
-4. Navigate into the root directory
-
-```sh
-  cd club_website
-```
+## Computer Science Club Website
+#### North Seattle College
 
-6. Install required packages
-    ```sh
-     npm install
-    ```
-7. Start server
-    ```sh
-     npm start
-    ```
+This is the website for the Computer Science Club at North Seattle. Besides the website this project exists as a place where people new to Github, Computer Science, and building applications can get involved. 
 
-<p align="right">(<a href="#top">back to top</a>)</p>
+### Contributing
 
-<!-- ROADMAP -->
+Do not be intimidated or dissuaded if you have little experience with any of the tools or languages involved in this project. There are multiple ways to contribute including editing documentation, design, feature requests, logging bugs, etc. 
 
-## Roadmap
+If you are a student at North Seattle College in any field we encourage you contribute! Please see [contribution guidelines for this project](CONTRIBUTING.md) for a full outline of how to get involved.
 
-See the [open issues]("GITHUB LINK"/issues) for a full list of proposed features (and known issues).
+#### Prerequisites
 
-<p align="right">(<a href="#top">back to top</a>)</p>
+##### MacOS
+- XCode Command Line Tools `xcode-select --install`.
+- If you already have the full Xcode installed, go to menu `Xcode -> Open Developer Tool -> More Developer Tools....`.
+- Install Node.js
 
-<!-- CONTRIBUTING -->
+##### Windows
+- Install current version of Python from Microsoft Store.
+- Install Visual Studio Code.
+    - Add the Microsoft C/C++ extention.
+- Install Node.js for Windows.
+- OR Install Windows Subsystem for Linux 2 and follow the linux instructions (not recommended unless familiar with WSL already) 
 
-## Contributing
+##### Linux
 
-Contributions are what make the open source community such an amazing place to learn, inspire, and create. Any contributions you make are **greatly appreciated**.
+- Install Python > 3.7
+- Install 'make'
+- A C/C++ compiler toolchain (the GNU Compiler Collection for example)
```

Comment:
No reason to duplicate the contributing file. Just add a brief message and link to it.